hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/gas/SystemContractGasCalculator.java

```diff
@@ -29,6 +30,9 @@
 public class SystemContractGasCalculator {
     private static final long FIXED_VIEW_GAS_COST = 100L;
 
+    // This represents the predefined gas price which is $0.000_000_0852 per unit of gas.
```

Comment:
maybe a bit more explanation like 852_000 = $0.000_000_0852 * 100(cents in dollars) * 100_000_000 (tiny cents in cents) * 1000 (Fee schedule units per tiny cents) or something like that.

Reply:
Done. I've adjusted it.